mitxpro/templatetags/render_notification.py

```diff
@@ -0,0 +1,16 @@
+"""Templatetags for rendering site notification"""
+
+from django import template
+from cms.models import SiteNotification
+
+register = template.Library()
+
+
+@register.inclusion_tag("../templates/partials/notifications.html", takes_context=True)
+def render_notification(context):
+    """ Return request context and notification(s)."""
+
+    return {
+        "notification": SiteNotification.objects.all().last(),
```

Comment:
`SiteNotification.objects.last()` would not be the same ?

Reply:
question: So we will show the last added notification? Are we checking for either is it active ? 
so all the other notifications are obsolete ?

Reply:
This is an assumption right now. Future work which is out of scope for now
- date range between which the content should be displayed
- ensure that only one site wide notification can be displayed at once.

At this point, I am just showing the latest notification.